### ISSUE TITLE
Does seq2seq punctuation restoration need a special loss？

### ISSUE THREAD

**Issue by yuboona** [choko/acoustic_punctuation#2]
**Hello，I wrote a seq2seq model with Pytorch to do punctuation restoration. And there is a question** 

My model:
- architechture
   - encoder: GRU
   - decoder: GRU with attention 
   - loss: Cross Entropy after a softmax
- output label
   - 5 types
   - (1) ',' (2) '.' (3) '!' (4) '?' (5) 'blank'

But after experiments, I found my model only learning to pridict 'blank'. The model tends to predict all input words to be a 'blank'.  What can I do to fix this? A special loss or other thing?
